docs/docusaurus/docs/reference/learn/integrations/dbt_tutorial.md

```diff
@@ -0,0 +1,418 @@
+- Right click on **Servers** in the top left corner and choose Register > Server
+- Name it postgres
+- Click on the Connection tab
+    - Host name/address: database (refers to the database docker service)
+    - Username and password: postgres (defined in Docker Compose under database service)
+- Save the connection
+
+![Create a PostgreSQL server](./dbt_tutorial/create_psql_server.png)
+
+You will now be able to see your empty database when you look under the public schema > tables:
+
+![Empty PostgreSQL tables](./dbt_tutorial/empty_psql_tables.png)
+
+## 5. Populate the Database 
+The dbt project that is included from GitHub has some initial datasets in the form of [seed files](https://docs.getdbt.com/docs/build/seeds). You will import these seed files into the database as tables and then use these tables as the data source for the rest of our dbt models.
+
+Open the dbt Docker container’s terminal via running the following command in your project’s terminal:
+```bash
+docker exec -it dbt bash -l
+```
+
+:::note
+Here you are using the `docker exec` command to open an interactive terminal inside the container using bash as the shell. The -l flag is important here because it tells the shell to be a login shell, which automatically sources the .bashrc file in the container, which in turn exports the dbt environment variables. See the dbt.Dockerfile for more info.
+:::
+
+Next, run the `dbt deps` command to install dependencies. Then, copy the seed data into the seeds folder by running `cp jaffle-data/*.csv` and run the `dbt seed` command to import the data into the PostgreSQL database. By default, it will import the data into a new schema called `jaffle_shop_raw`. This step might take a few minutes to complete.
+
+```bash
+dpt deps
+cp jaffle-data/*.csv seeds
+dbt seed
+```
+
+![dbt seed](./dbt_tutorial/dbt_seed.png)
+
+In pgAdmin you can now see the newly created tables in the new schema.
+
+![Raw tables created](./dbt_tutorial/raw_tables.png)
+
+Run the rest of the dbt models by running the command below in the dbt service command line:
+```bash
+dbt run
+```
+
+You will see the models have been created successfully as tables and views within the public schema:
+
+![Seeded tables](./dbt_tutorial/seeded_tables.png)
+
+To exit the dbt Docker container's terminal, run the following command:
+```bash
+exit
+```
+
+## 6. Use Great Expectations to create data quality tests
+In the next section of this tutorial, you will follow the [Getting Started](https://docs.greatexpectations.io/docs/oss/get_started/get_started_with_gx_and_sql) guide to run GX with SQL within a Docker container. This guide describes the process using Jupyter Notebook, although you can use any Python development tool you prefer.
+
+### Initialize the Jupyter Notebook Server
+Run the following command within the project directory to start the Jupyter Notebook server:
+```bash
+docker exec great-expectations jupyter notebook --allow-root --no-browser --ip=0.0.0.0
```

Comment:
I think you need to instruct them to install jupyter first or this command won't work